timon/probes.py

```diff
@@ -403,31 +418,25 @@ def check_match_rule(rule):
 
         rule_type, match = check_match_rule(rule)
         if rule_type:
+            if rule_type == "NONE":
+                return False
+            if rule_type == "DEFAULT":
+                return True
             fields = match.groups()[0].split(".")
             rule_val = match.groups()[1:]
-            val = minibelt.get(rslt, keys=fields)
+            val = minibelt.get(rslt, *fields)
             if rule_type == "equal_rule":
                 return str(val) == rule_val[0]
-            elif rule_type == "greater_rule":
+            if rule_type == "greater_rule":
```

Comment:
Why you changed all elif to if ?
Cannot have multiple rules at the same time so I don't understand why you do that ..

Reply:
coding style guide lines suggest to use if... elif only if necessary.

```
if bla
   return X
elif blo
    return Y
```

doesn't need the elif, as there is a return


same as

```
if a:
   return X
else:
   something_else
```

can / should be written as
```
if a
    return X
something_else
```